### PR TITLE
Adding information on new features for DCAT 3 and prioritization

### PR DESCRIPTION
**Pull request by riccardoAlbertoni**
This PR integrates the DCAT 3 Explainer adding information on new features for DCAT 3 and prioritization as required #1530. 

Please check if other important matters need to be mentioned. See [current draft](https://github.com/w3c/dxwg/blob/e3afedfe6f6c4c0200f081ffd8df5fc191e8e8c7/docs/explainer.md)

### CODE REVIEW

**Comment by riccardoAlbertoni** [comment]
Please consider the current version of the explainers. It should be coherent with the discussion we had in yesterday's DCAT call, but feel free to suggest changes or propose to omit any information if you think the changes are too lengthy.

We will reply about the usage of RDF lists separately.

**Review by andrea-perego — APPROVED** [review]

File: docs/explainer.md [hunk 1]
```diff
@@ -12,7 +12,31 @@ The original DCAT vocabulary was developed and hosted at the Digital Enterprise
...
+
+## Prioritization of the issues /requirements to address in DCAT 3
+The prioritization was made considering the [requirements left out from the second round of standardization](https://github.com/w3c/dxwg/issues?q=label%3Adcat++closed%3A%3E2020-02-01+label%3Arequirement+) and the feedback received at the time.
+
+We organized the issues on different [GitHub Projects](https://github.com/w3c/dxwg/projects?query=is%3Aopen&type=classic), splitting the effort into Thematically-homogeneous sprints. 
+
+  Among the sprints, we decide to prioritize [DCAT Sprint: Dataset Series](https://github.com/w3c/dxwg/projects/10) and [DCAT Sprint: Versioning](https://github.com/w3c/dxwg/projects/9) as
+- they counted a quite conspicuous number of open issues, 14 and 18 open issues, respectively
```
Comment (agbeltran) [inline docs/explainer.md]: did you mean conspicuous or copious?

File: docs/explainer.md [hunk 2]
```diff
@@ -12,7 +12,31 @@ The original DCAT vocabulary was developed and hosted at the Digital Enterprise
...
+- they seemed to be two core building blocks whose stabilization could simplify addressing other aspects
+- they were relevant for different communities, from governmental to open science.
+
+Of course, we didn't work only on Dataset series and Versioning,
```
Comment (agbeltran) [inline docs/explainer.md]: I would change this sentence to: "In addition to Dataset series and Versioning, we also worked on other issues".

**Review by agbeltran — CHANGES_REQUESTED** [review]

**Comment by agbeltran** [comment]
thanks @riccardoAlbertoni - I pushed a couple of minor changes, and left a couple of text suggestions for your consideration.

File: docs/explainer.md (hunk 1 above)
Comment (riccardoAlbertoni) [reply docs/explainer.md]: Both are fine for me. I'll use copious as it seems that you think it is better.

Comment (riccardoAlbertoni) [inline docs/explainer.md]:
```suggestion
- they counted a quite copious number of open issues, 14 and 18 open issues, respectively
```

File: docs/explainer.md (hunk 2 above)
Comment (riccardoAlbertoni) [inline docs/explainer.md]: Fine to me
```suggestion
In addition to Dataset series and Versioning, we also worked on other issues,
```

**Review by riccardoAlbertoni — COMMENTED** [review]
Accepting Alejandra's suggestions

**Comment by riccardoAlbertoni** [comment]
@agbeltran  I have accepted your requests for changes.
Please, unlock the PR merging if you don't have any other observations.

File: docs/explainer.md [hunk 3]
```diff
@@ -12,7 +12,31 @@ The original DCAT vocabulary was developed and hosted at the Digital Enterprise
...
+The prioritization was made considering the [requirements left out from the second round of standardization](https://github.com/w3c/dxwg/issues?q=label%3Adcat++closed%3A%3E2020-02-01+label%3Arequirement+) and the feedback received at the time.
+
+We organized the issues on different [GitHub Projects](https://github.com/w3c/dxwg/projects?query=is%3Aopen&type=classic), splitting the effort into Thematically-homogeneous sprints. 
+
+  Among the sprints, we decide to prioritize [DCAT Sprint: Dataset Series](https://github.com/w3c/dxwg/projects/10) and [DCAT Sprint: Versioning](https://github.com/w3c/dxwg/projects/9) as
+- they counted a quite copious number of open issues, 14 and 18 open issues, respectively
+- they seemed to be two core building blocks whose stabilization could simplify addressing other aspects
+- they were relevant for different communities, from governmental to open science.
+
+In addition to Dataset series and Versioning, we also worked on other issues,
```
Comment (riccardoAlbertoni) [inline docs/explainer.md]:
```suggestion
In addition to Dataset series and Versioning, we also worked on other issues.
```

**Review by agbeltran — APPROVED** [review]

**Pull request merged**